bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/common-base-channel-types.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="baseType">
+		<item-type>Number:Energy</item-type>
+		<label>BASE</label>
```

Comment:
From the usability perspective it would be great if you write the same text for the parameter here as in the readme.

Reply:
Ok (README.md file to update)

Reply:
@fwolter @nokyyz Can you explain? Do I need to copy the description field from the table in the readme to the `<description>` field in the xml? Or is it a problem of uppercase/lowercase for the label?

Reply:
Yes, copying the description field from the readme into `<description>` would be great.

Reply:
Thanks! nokyyz remark about updating readme confused me.